src/Microsoft.Azure.WebJobs.Host/Bindings/IValueProvider.cs

```diff
@@ -19,7 +20,7 @@ public interface IValueProvider
         /// Gets the value.
```

Comment:
nit: Can you update the help text for these similar to what we use in other places for task returning values, e.g. IBinding.BindAsync, etc. E.g. "A task that returns the value".